server/server.go

```diff
@@ -91,7 +92,7 @@ func New(c *Config) (*Server, error) {
 		githubTimeout = 10 * time.Second
 	}
 
-	userAgent := fmt.Sprintf("%s/%s", c.Options.AppName, version.GetVersion())
+	userAgent := fmt.Sprintf("policy-bot/%s", version.GetVersion())
```

Comment:
Using the app name here requires creating a separate app client and didn't seem worth it. I think `policy-bot` is an appropriate user agent in most cases, even if the app is registered with a different name.